### DISCUSSION TITLE
Welcome to Plugin.FirebasePushNotifications Discussions!

### DISCUSSION THREAD

**Discussion by thomasgalliker** [thomasgalliker/Plugin.FirebasePushNotifications#18] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by efimomax** [comment]
Hello! Very useful plugin. Thank you for it.

One question: Is it possible to get delivered on device push messages, but not tapped?
For example, if i open app from background i want to analize if the some messages arrived and stay in queue in system (also in status bar)

Thanks

**Reply by thomasgalliker** [reply]
Thank you. Yes, I have code for this operation. I‘ll extend the interface IFirebaseNotifications.

**Reply by thomasgalliker** [reply]
Something like this?
https://github.com/thomasgalliker/Plugin.FirebasePushNotifications/issues/26

**Reply by efimomax** [reply]
YES exactly! 
As I said, there is a need to check from the app "if there are any push notifications that came but were not clicked". And also to clear not the whole queue, but a specific one by ID. :)

Thanks a lot!

**Reply by thomasgalliker** [reply]
You‘re talking about different problems. I checked if it is possible to return a list of delivered but not opened notifications: it is possible, but I can only see a list of notification Ids with all Ids=0. Android does not give access to the notification payload. So, if you have an idea how you‘d like to solve it, let me know.

**Reply by thomasgalliker** [reply]
Comment in #26
